apps/parse/lib/parse/stop_times.ex

```diff
@@ -6,7 +6,6 @@ defmodule Parse.StopTimes do
   import NimbleParsec
   import Parse.Helpers
 
-  # credo:disable-for-lines:4 Credo.Check.Refactor.PipeChainStart
```

Comment:
I can revert this. I wasn't sure if anyone had explicitly wanted it to be enabled, since I believe it is by default, and I personally find it has enough edge cases to make it annoying even though I agree with the general principle (there ended up being 4 violations in `connecting_stops.ex` that I thought were all less readable if "corrected").

Reply:
> I wasn't sure if anyone had actively wanted it to be enabled, since I believe it is by default

For the record I was not correct about this; it is disabled by default, and placed in a "controversial and experimental" section of the config.